rocketmq-console/src/main/resources/application.properties

```diff
@@ -14,5 +14,9 @@ rocketmq.config.isVIPChannel=
 rocketmq.config.dataPath=/tmp/rocketmq-console/data
 #set it false if you don't want use dashboard.default true
 rocketmq.config.enableDashBoardCollect=true
+# rocketMQ server default admin account
+rocketmq.config.accessKey=rocketmq2
```

Comment:
not to set the ak/sk properties by default for backward compatible, namely, disable ACL by default

Reply:
If ak or sk is null, the acl configuration will not take effect and the acl functionality will be disabled.